mlrun/db/httpdb.py

```diff
@@ -2635,10 +2646,14 @@ def list_model_endpoints(
                                  `'now-[0-9]+[mhd]'`, where `m` = minutes, `h` = hours, and `'d'` =
                                  days), or 0 for the earliest time.
         :param top_level: if true will return only routers and endpoint that are NOT children of any router
-        :param uids: if passed will return ModelEndpointList of endpoints with uid in uids
+        :param uids: if passed will return `ModelEndpointList` of endpoints with uid in uids
         """
 
         path = f"projects/{project}/model-endpoints"
+
+        if labels and isinstance(labels, dict):
+            labels = json.dumps(labels)
```

Comment:
Why use json here? If you get a dict, simply create a list of `key=value` strings:
```
labels = [f"{key}={value}" for key, value in dict.items()]
```
Not to mention, the `labels` parameter is hinted at and documented as getting a list of strings, so where did the `dict` get here?